sdcm/cluster.py

```diff
@@ -1784,7 +1784,8 @@ def install_package(self, package_name: str) -> None:
         self.remoter.sudo(f'{pkg_cmd} install -y {package_name}')
 
     def is_apt_lock_free(self) -> bool:
-        return self.remoter.sudo("lsof /var/lib/dpkg/lock", ignore_status=True).ok
+        result = self.remoter.sudo("lsof /var/lib/dpkg/lock", ignore_status=True)
+        return result.return_code == 1
```

Comment:
It's more typical for SCT' code to use `.exit_status` attribute. Is there any reason to use `.return_code` instead?

Reply:
No, it was only my preference. I changed it to `exit_status`.